Bibliographie

### Article 1: Cell-type-specific gene expression profiling in adult Mouse brain reveals Normal and disase-state signatures
***
Method less stressful for cells to dissec & investigate  cellular specificty trasncriptome & chip-seq. => Opposite epigenetic govern transcriptionnal regulation of neurones & glial cells -> compensatory mechanisms.  
HD up gene: more represented in glia  
HD down genes more represented in neurones  

### Article 2: Huntington disease  
***
More prevalent in some pop: europ > asia > africa  
\> 36 CAG: desease  
27-35: depends of the person + de novo  
CAG repeat: gain of function for huntingtin  
Function of normal huntingtin are known  
Toxic gain of function -> critical conformational changes ?  
HTT exon 1 contain a N17 (17 AA n-ter) domain which is highly flexible and considered disordered protein  

### Article 3: The pathobiology of perturbed mutant Huntingtin protein-protein interactions in Huntington's diseases
***
Mutant HTT can lead to abnormal PPI (prot-prot interaction) associated leading to formation of aggregates 
Different forms of mHTT (mono, oligo, fibrilar) can convert into each other: enventually large inclusion bodies, forms can have different interaction protein  
wtHTT: multivalent hub interact w/ differnt prot  
mHTT & wtHTT: struturally distinct prot = diff. interaction profile ?  
Abnormal mHTT - cellular prot interaction:  
* Axonal transport
* Autophagy  
* mTOR signalling (Rhes enriched stratium, reduce HD patient)  
* MitoC fission (DRP1)  
* Gene transcription regulation 

**Gene trasncription**  
wtHTT bind transcription factors = gene expression regulator  
Binding of trascriptionnal activator such a SP1 & repressor such as RE-1  
mHTT: bind more strongly to SP1. mHTT repress SP1-dep transcription
