Bibliographie

### Article 1: Cell-type-specific gene expression profiling in adult Mouse brain reveals Normal and disase-state signatures
***
Method less stressful for cells to dissec & investigate  cellular specificty trasncriptome & chip-seq. => Opposite epigenetic govern transcriptionnal regulation of neurones & glial cells -> compensatory mechanisms.  
HD up gene: more represented in glia  
HD down genes more represented in neurones  

### Article 2: Huntington disease  
***
More prevalent in some pop: europ > asia > africa  
\> 36 CAG: desease  
27-35: depends of the person + de novo  
CAG repeat: gain of function for huntingtin  
Function of normal huntingtin are known  
Toxic gain of function -> critical conformational changes ?  
HTT exon 1 contain a N17 (17 AA n-ter) domain which is highly flexible and considered disordered protein  

### Article 3: The pathobiology of perturbed mutant Huntingtin protein-protein interactions in Huntington's diseases
***
Mutant HTT can lead to abnormal PPI (prot-prot interaction) associated leading to formation of aggregates 
Different forms of mHTT (mono, oligo, fibrilar) can convert into each other: enventually large inclusion bodies, forms can have different interaction protein  
wtHTT: multivalent hub interact w/ differnt prot  
mHTT & wtHTT: struturally distinct prot = diff. interaction profile ?  
Abnormal mHTT - cellular prot interaction:  
* Axonal transport
* Autophagy  
* mTOR signalling (Rhes enriched stratium, reduce HD patient)  
* MitoC fission (DRP1)  
* Gene transcription regulation 

**Gene trasncription**  
wtHTT bind transcription factors = gene expression regulator  
Binding of trascriptionnal activator such a SP1 & repressor such as RE-1  
mHTT: bind more strongly to SP1. mHTT repress SP1-dep transcription
Abnormal homotypic mHTT interaction: aggregates that perturb cellular functions 
Aggregates: sequester quality control prot (ubiquitin, chaperone, 26S proteasome)  

```
"Striatal selectivity of HD may be explained by binding of mHTT to proteins that are predominantly expressed in this brain region"
```
### Article 4: Thèse de Caroline Lotz: "Rôle des altérations transcriptionnelles et épigénétiques dans le déficits comportementaux de la maladie de Huntington"
**Dérégulation transcriptionnelle**  
Amplitude déréulation transcriptionnelles dépend du nb ce CAG. Particulièrement étendue dans le striatum. Gène d'identité des neurones striataux: largement sous exprimés chez HD: *Drd1, Drd2, Darpp32, PPP1R1B, Pde10a*  : fonction neuronaux = C. NEURONAUX  
Sur-exprimé: moins robuste, fonctions adhésion Cellulaire.   = C. GLIALE  
Création de mécanismes compensatoires  
Interaction mHTT avec TF. (comme la CBP)  

**Altérations épigénétiques**
H3K27ac spécifiquement enrichie au niveau des enhancers actifs  
Les genes de neuronnes sous régulés en HD, sont régulés par des super-enhancers = des enhancers vastes enrichis en H3K27ac qui couvrent les gènes qu'ils régulent.  
Baisse de H3K27ac à ces enhancers (et pol II) chez souris R6.  
H3K4me3 associé promoteurs actifs, participent sous reg de l'identité neuronal -> ces gènes ont une signature en H3K4me3 large similaire a H3K26ac  
Moins de H3K27ac -> lié à séquestration de CBP ? (car substrat)  
H3K27me3: marque repression à activité augmenté dans HD.